modules/glm/1.0.1/patches/bazelize.patch

```diff
@@ -0,0 +1,187 @@
+diff --git a/BUILD.bazel b/BUILD.bazel
+new file mode 100644
+index 000000000..c8f1e44e0
+--- /dev/null
++++ b/BUILD.bazel
+@@ -0,0 +1,11 @@
++cc_library(
++    name = "glm",
++    hdrs = glob([
++        "glm/**/*.h",
++        "glm/**/*.hpp",
++    ]),
++    include_prefix = "glm",
++    includes = ["."],
++    textual_hdrs = glob(["glm/**/*.inl"]),
++    visibility = ["//visibility:public"],
++)
+diff --git a/MODULE.bazel b/MODULE.bazel
+new file mode 100644
+index 000000000..c9a1cca93
+--- MODULE.bazel
++++ MODULE.bazel
+@@ -0,0 +1,6 @@
++module(
++    name = "glm",
++    version = "1.0.1",
++)
++
++bazel_dep(name = "rules_cc", version = "0.0.9")
+diff --git a/test/MODULE.bazel b/test/MODULE.bazel
+new file mode 100644
+index 000000000..0b7e61a35
+--- /dev/null
++++ b/test/MODULE.bazel
+@@ -0,0 +1,5 @@
++bazel_dep(name = "glm", version = "1.0.1")
++local_path_override(
++    module_name = "glm",
++    path = "..",
++)
+diff --git a/test/WORKSPACE.bazel b/test/WORKSPACE.bazel
```

Comment:
This line seems to cause a failure, since there is no such file:

Error in patch: Error applying patch /home/fjsinis/.cache/bazel/_bazel_fjsinis/7f72cbe255b2cadb8f59e8a4302e9c4c/external/glm~/.tmp_remote_patches/bazelize.patch: Cannot find file to patch (near line 44), old file name (a/test/WORKSPACE.bazel) doesn't exist, new file name (b/test/WORKSPACE.bazel) doesn't exist.